pkgs/servers/exhibitor/default.nix

```diff
@@ -9,7 +9,7 @@ stdenv.mkDerivation rec {
     sha256 = "07vikhkldxy51jbpy3jgva6wz75jksch6bjd6dqkagfgqd6baw45";
     rev = "5fcdb411d06e8638c2380f7acb72a8a6909739cd";
   };
-  mavenDependenciesSha256 = "00r69n9hwvrn5cbhxklx7w00sjmqvcxs7gvhbm150ggy7bc865qv";
+  mavenDependenciesSha256 = "sha256-oGd8PFF1Z9BElNsgqbC/llQm8j/T+T01VwvSw3tTU0E=";
```

Comment:
@wegank From https://github.com/soabase/exhibitor:

> This repository has been archived by the owner on Apr 23, 2020. It is now read-only.

That's three years ago. Is that something that would be removed eventually? There's a whole service that depends on this in NixOS, though, if I understand this correctly: https://github.com/NixOS/nixpkgs/blob/master/nixos/modules/services/misc/exhibitor.nix#L382

Reply:
Dropped.